web/util/date.ts

```diff
@@ -30,6 +26,22 @@ export const formatTimeDuration = (start: Date, duration: number) => {
   return `PT${hours ? hours + 'H' : ''}${minutes ? minutes + 'M' : ''}`;
 };
 
+const differingMeridiem = (d1: Date, d2: Date) =>
+  d1.getHours() >= 12 !== d2.getHours() >= 12;
+
+export const formatTimeRange = (
+  start: Date,
+  duration: number,
```

Comment:
Why would this function reverse it again? The only place I see it using `duration` is to compute an `end` variable.

Reply:
Sorry, poorly formulated. 

We receive a `start` Date and a `Duration` from Sanity. I believe it's more convenient to expose a util function to use those, rather than requiring callers to convert the `duration` into a Date.